docs/api/topics/firefox_os_addons.rst

```diff
@@ -58,8 +58,8 @@ Add-on
     :type download_url: string
     :param name: The add-on name.
     :type name: string|object
-    :param manifest_url: The (absolute) URL to the mini-manifest for that add-on.
-    :type manifest_url: string
+    :param mini_manifest_url: The (absolute) URL to the mini-manifest for that add-on.
+    :type mini_manifest_url: string
```

Comment:
@ngokevin , @chuckharmston : how do you feel about this change ? I think it's less confusing that way. Since the 2 formats are completely different I figured it was probably best to make sure we don't confuse them.